apps/test/unit/templates/ButtonTest.js

```diff
@@ -173,4 +173,25 @@ describe('Button', () => {
     assert(icon);
     assert.equal(icon.props().icon, 'lock');
   });
+
+  it('supports the download attribute, but only for anchors', () => {
+    let wrapper = shallow(
+      <Button __useDeprecatedTag download href="/foo/bar" text="Click me" />
+    );
+    assert.equal(wrapper.find('a').prop('download'), true);
+
+    wrapper = shallow(
```

Comment:
Mind splitting these into two tests? One with the anchor and the other without?

The two scenarios seem pretty separate, but happy to have my mind changed :)

Reply:
I thought about writing two tests, but decided to be lazy :) I'll go ahead and un-lazy this